TELCODOCS-2043 - CNF-13766 - unnumbered BGP peering (#89663)
openshift/openshift-docs: modules/nw-metallb-example-bgppeer.adoc

  peerASN: 64500
  myASN: 64500
----

[id="nw-metallb-example-unnumbered-bgp-peering_{context}"]
== Example: Specify BGP peers for unnumbered BGP peering

include::snippets/technology-preview.adoc[]

To configure unnumbered BGP peering, specify the interface in the `spec.interface` field.

[source,yaml]
----
apiVersion: metallb.io/v1beta2
kind: BGPPeer
metadata:
  name: peer-unnumber
  namespace: metallb-system
spec:
  myASN: 64512
  ASN: 645000
  interface: net0
----
[NOTE]
====

> To use the `interface` field, there must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).

Reviewer: Again here its not a traditional p2p but it needs to be layer 2. The most important point is that there can only be one BGP connection per interface.

  peerASN: 64500
  myASN: 64500
----

[id="nw-metallb-example-unnumbered-bgp-peering_{context}"]
== Example: Specify BGP peers for unnumbered BGP peering

include::snippets/technology-preview.adoc[]

> To configure unnumbered BGP peering, specify the interface in the `spec.interface` field.

Reviewer 2: ```suggestion
To configure unnumbered BGP peering, specify the interface in the `spec.interface` field by using the following example configuration:
```

[source,yaml]
----
apiVersion: metallb.io/v1beta2
kind: BGPPeer
metadata:
  name: peer-unnumber
  namespace: metallb-system
spec:
  myASN: 64512
  ASN: 645000
  interface: net0
----
[NOTE]
====

> To use the `interface` field, there must be a point-to-point, layer 2 connection between the two BGP peers.

Reviewer 2: ```suggestion
To use the `interface` field, you must establish a point-to-point, layer 2 connection between the two BGP peers.
```

 considering the connection can be set by users
